packages/database/prisma/schema.prisma

```diff
@@ -212,6 +212,27 @@ model ScannerJob {
   @@index([packageId])
 }
 
+/// A system issue.
```

Comment:
This comment could be expaned to make it clearer what "a system issue" actually is.